### DISCUSSION TITLE
Error: Io(Error { kind: UnexpectedEof, message: "failed to fill whole buffer" })

### DISCUSSION THREAD

**Discussion by tensorajack** [zkonduit/ezkl#799] [Q&A]
I am running this command from the cli:

ezkl gen-witness -D temp/input_0.json -M deployment_layer/model_0/model.compiled -V deployment_layer/model_0/vk.key -P deployment_layer/model_0/kzg.srs

My input file is a json file: 

{"input_data": [[-0.6377028034728642, -0.8591870486368947, -0.643772047297958, 0.5050098305775228, 0.9551093480884891]], "input_shapes": [[1]]}

Is anyone able to tell me how to either troubleshoot or fix the error received:

io error: failed to fill whole buffer

**Comment by alexander-camuto** [comment]
did we resolve this in TG ?

**Reply by tensorajack** [reply]
I guess in a way, although the answer was to go back to version 10.2.7. Not sure if you to open an actual issue for the newer versions to get them fixed so they work?
